rt/features/logging/src/main/java/org/apache/cxf/ext/logging/LoggingFeature.java

```diff
@@ -117,4 +117,50 @@ public void setLogMultipart(boolean logMultipart) {
     public void setVerbose(boolean verbose) {
         setSender(verbose ? new Slf4jVerboseEventSender() : new Slf4jEventSender());
     }
+
+    /**
+     * Set binary media types for Logging In Interceptor.
+     * Contet for these types was not logged.
```

Comment:
"Content" not "Contet" (Same below as well)

Reply:
Correct this